platform-dependent-components/problem-solver/cxx/sections-module/keynodes/sections_keynodes.hpp

```diff
@@ -5,52 +5,36 @@
  */
 
 #pragma once
-
-#include "sc-memory/sc_addr.hpp"
-#include "sc-memory/sc_object.hpp"
-
-#include "generated/sections_keynodes.generated.hpp"
+#include <sc-memory/sc_keynodes.hpp>
 
 namespace sectionsModule
 {
-class SectionsKeynodes : public ScObject
+class SectionsKeynodes : public ScKeynodes
 {
-  SC_CLASS()
-  SC_GENERATED_BODY()
-
 public:
-  SC_PROPERTY(Keynode("action_get_decomposition"), ForceCreate)
-  static ScAddr action_get_decomposition;
+  static inline ScKeynode const action_get_decomposition{"action_get_decomposition"};
```

Comment:
Please, add types for all keynodes here

Reply:
static inline ScKeynode const action_get_decomposition{"action_get_decomposition", ScType::ConstNodeClass};